src/xenia/kernel/xam/xam_input.cc

```diff
@@ -20,6 +20,8 @@ namespace xe {
 namespace kernel {
 namespace xam {
 
+DEFINE_bool(enable_vibration, true, "Enable controller vibration", "HID");
```

Comment:
`bool` implies "enable", there's no need for the "enable" word. Most config variable descriptions also end with a period.